src/Business Foundation/App/NoSeriesCopilot/src/Utilities/NoSerCopNotificManager.Codeunit.al

```diff
@@ -0,0 +1,33 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.Foundation.NoSeries;
+
+codeunit 348 "No. Ser. Cop. Notific. Manager"
```

Comment:
Move these into the `No Series Copilot Impl`. Doesn't make sense for a separate codeunit for these functions.